content/docs/reactivesearch/vue-searchbox/quickstart.md

```diff
@@ -41,11 +41,11 @@ The following example renders an autosuggestion search bar(`search-component`) w
 
 ```html
 <template>
```

Comment:
@mohdashraf010897 We don't want format changes here. You might have used the prettier to format it. Remove formatting changes and just update the content.